docs/cli.md

```diff
@@ -29,17 +34,303 @@ Common Flags
+
+### oc status
+
+This shows a high level overview of the current project.
+
+See also [`oc describe`](#oc-describe) and [`oc get`](#oc-get).
+
+### oc project
+
+This displays the current project, or switches to another one.
+
+For example:
+
+```
+# Switch to the myapp project
+$ oc project myapp
+
+# Display the project currently in use
+$ oc project
+```
+
+## Build and Deploy Commands
+
+### oc start-build
+
+This manually starts a build, using either the specified buildConfig or a build name with the `--from-build` option.
+There is also the option of streaming
+the logs of the build if the `--follow` flag is specified.
+
+```
+$ oc start-build ruby-sample-build
+$ oc start-build --from-build=ruby-sample-build-275d3373-c252-11e4-bc79-080027c5bfa9
+$ oc start-build --from-build=ruby-sample-build-275d3373-c252-11e4-bc79-080027c5bfa9 --follow
+```
+
+See also [`oc new-build`](#oc-new-build) and [`oc new-app`](#oc-new-app).
+
+### oc build-logs
+
+This retrieves the logs from a Build container.
+It allows you to
+debug broken Build.
+If the build is still running, this streams the logs from the container to console.
+Use `oc get builds` to obtain a list of builds.
+
+```
+$ oc build-logs rubyapp-build
+```
+
+### oc deploy
+
+This views, starts, cancels or retries deployments.
+The general form is:
+
+```
+$ oc deploy <deployment-config> [options]
+```
+
+If invoked without options, display the latest deployment for the specified *deployment-config*.
+For example:
+
+```
+$ oc deploy database
+```
+
+Flag `--latest` starts a deployment.
+Flag `--retry` retries the latest failed deployment.
+Flag `--cancel` cancels the in-progress deployment.
+The following example shows how to cancel the deployment mentioned in the preceding example:
+
+```
+$ oc deploy database --cancel
+```
+
+### oc rollback
+
+This reverts the pod and container configuration back to a previous deployment.
+Scaling and trigger settings are normally left as-is.
+Flag `--dry-run` displays what the rollback *would do* but does not perform the rollback.
+Flags `--change-triggers` and `--change-strategy` include the previous deployment's triggers and strategies, respectively, in the rollback.
+Flag `--change-scaling-settings` includes the previous deployment's replication controller replica count and selector in the rollback.
+
+If specified, option `--output <format>` inhibits the rollback and instead displays updated deployment configuration in the specified *format*:
+
+* `json`
+* `yaml`
+* `template`
+* `templatefile`
+
+With `--output template`, option `-t <template-string>` specifies a template string.
+With `--output templatefile`, option `-t <filename>` specifies a template file.
+
+Here are two examples:
+
+```
+# Perform a rollback.
+$ oc rollback deployment-1
+
+# Perform the rollback "manually" by piping back to "oc update".
+$ oc rollback deployment-1 --output=json | oc update dc deployment -f -
+```
+
+See also [`oc update`](#oc-update).
```

Comment:
@kargakis Thanks for catching this.  See: https://github.com/tnguyen-rh/origin/commit/d184f4411efafe3363113da06e1415481da67fcf